apps/api/src/app/widgets/pipes/limit-pipe/limit-pipe.ts

```diff
@@ -0,0 +1,36 @@
+import { ArgumentMetadata, BadRequestException, Injectable, PipeTransform } from '@nestjs/common';
+
+@Injectable()
+export class LimitPipe implements PipeTransform {
+  private readonly minInt: number;
+  private readonly maxInt: number;
+  private readonly isOptional: boolean = false;
+  constructor(minInt: number, maxInt: number, isOptional?: boolean) {
+    this.minInt = minInt;
+    this.maxInt = maxInt;
+    if (isOptional != null) {
+      this.isOptional = isOptional;
+    }
+  }
+  transform(value: number | undefined, metadata: ArgumentMetadata) {
+    if (this.isOptional && value == null) {
+      return value;
+    }
+
+    if (!this.isOptional && value == null) {
```

Comment:
loose equality comparison is bad practice... I think it's hard to spot that it's loose and I feel like not everyone might remember that `undefined == null` is also true...

Reply:
Do you mean `value === null || value === undefined` will be more easy to read?

Reply:
like it's more text, but it has a clear meaning... with `value == null` I might not notice that it's a `==` and not `===`